packages/x-components/src/x-modules/scroll/components/scroll-to-top.vue

```diff
@@ -15,12 +15,14 @@
 
 <script lang="ts">
   import Vue from 'vue';
-  import { Component, Prop, Watch } from 'vue-property-decorator';
-  import { XOn } from '../../components/decorators/bus.decorators';
-  import { WireMetadata, XEventsTypes } from '../../wiring';
-  import BaseEventButton from '../base-event-button.vue';
-  import { NoElement } from '../no-element';
-  import { ScrollDirection } from './scroll.types';
+  import { Component, Prop } from 'vue-property-decorator';
+  import { State, xComponentMixin } from '../../../components';
+  import BaseEventButton from '../../../components/base-event-button.vue';
+  import { NoElement } from '../../../components/no-element';
+  import { Dictionary } from '../../../utils';
+  import { XEventsTypes } from '../../../wiring';
+  import { ScrollComponentState } from '../store';
+  import { scrollXModule } from '../x-module';
 
   /**
    * Renders a button with a default slot.
```

Comment:
`Renders a button with a default slot and an animation.` ?

Reply:
But there is not default animation, right? You have to pass in the animation.

Reply:
I agree with @LuisMartinez15 The animation optional. However I have simplified the documentation, too much detail.